media-libs/opencv/opencv-4.10.0.ebuild

```diff
@@ -0,0 +1,1061 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+inherit cuda java-pkg-opt-2 java-ant-2 cmake-multilib flag-o-matic python-r1 toolchain-funcs virtualx
```

Comment:
You could help getting rid of `java-ant-2` which needs some more change further down in the ebuild.
As an example see the diff [arduino-1.8.19](https://github.com/gentoo/gentoo/blob/master/dev-embedded/arduino/arduino-1.8.19.ebuild) -> [arduino-1.8.19-r1](https://github.com/gentoo/gentoo/blob/master/dev-embedded/arduino/arduino-1.8.19-r1.ebuild)

inherit cuda java-pkg-opt-2 cmake-multilib flag-o-matic python-r1 toolchain-funcs virtualx

Reply:
Done in c0b97f4